src/database/schema/ChatSchema.ts

```diff
@@ -4,7 +4,11 @@ import {v4 as uuid} from 'uuid';
 import BaseDbSchema from './BaseDbSchema';
 import ChannelListSchema from './ChannelListSchema';
 import UserSchema from './UserSchema';
-import {DELETED_MESSAGE_TEXT, MESSAGE_TYPE_DELETED} from '../../utils/constants';
+import {
+  DELETED_MESSAGE_TEXT,
+  MESSAGE_TYPE_DELETED,
+  MESSAGE_TYPE_REPLY_PROMPT
+} from '../../utils/constants';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement has been modified to include a new constant `MESSAGE_TYPE_REPLY_PROMPT`. This change is necessary for the logic introduced in the function `isSavingAttachmentAllowed`.
```diff
- import {DELETED_MESSAGE_TEXT, MESSAGE_TYPE_DELETED} from '../../utils/constants';
+ import {
+   DELETED_MESSAGE_TEXT,
+   MESSAGE_TYPE_DELETED,
+   MESSAGE_TYPE_REPLY_PROMPT
+ } from '../../utils/constants';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->